setup.py

```diff
@@ -105,12 +105,13 @@
         'lmfit>=1.0,<2',
         'requests>=2.21.0,<3',
         'colorama>=0.4.1,<1',
-        'pandas>=1.0,<2',
-        'scipy==1.4.1',
+        'pandas>=1.0',
+        'scipy>=1.4.1',
         'six',
         'piradon',
         'matplotlib',
         'h5py',
+        'sympy==1.4.2'

```

Comment:
Why pin this at `1.4.2` when 1.8 is available. Should it not rather be `sympy>=1.8,<2`?

Reply:
Because in my course (PHS1022) they are asking specifically for that version as later version have issues with Numpy apparently. There is a work around but I am following the standard mentioned.

Reply:
I had a chat with PHS1022 organisation and they will adapt to the newer version from next year. So I will make the change but first merge after the end of semester.